core/config.cpp

```diff
@@ -77,15 +78,27 @@ bool Config::init_with_args(int ac, char** av, const std::vector<std::string>& c
                                      "Worker information.\nFormat is '%worker_hostname:%thread_number'.\nUse "
                                      "colon ':' as separator.");
 
+    po::options_description optional_options("Optional options");
```

Comment:
I think this may not be suitable to register an option here, as this is used for worker only, not master.